cassiopeia/solver/HybridSolver.py

```diff
@@ -326,7 +331,7 @@ def apply_bottom_solver(
             missing_state_indicator=cassiopeia_tree.missing_state_indicator,
             priors=cassiopeia_tree.priors,
         )
-        self.bottom_solver.solve(subtree)
+        self.bottom_solver.solve(subtree, logfile=logfile)
```

Comment:
This actually occurred to me as well! Right now it works only because we've tried using the `ILPSolver` as the bottom-solver. I'll add `logfile` as an argument for `solve` so this should work across the board.